docs/application/dotnet/guides/nui/resources.md

```diff
@@ -3,14 +3,47 @@
 NUI provides several ways to handle resource images.
 This tutorial describes the use of Image resources in NUI.
 
-## Overview
-
-Resources in NUI can apply Images or three-dimensional Models.
-
-The common method to access these resources is through Visuals.
-Controls use these visuals to display what is required.
-
-See [ImageView tutorial](imageview.md).
+## Image handling methods
+
+The two basic methods of handling image resources in NUI are as follows:
+ 
+1. `Visuals` are the main building block for UI components and provide reusable rendering logic that is controlled using properties. For more information, see [Visuals tutorial](visuals.md) page:
+    ```
+    //Define unique visual index.
+    const int IMAGE_VISUAL_INDEX = 100001;
+
+    //Create property map
+    PropertyMap imagePropertyMap = new PropertyMap();
+    imageVisual.Add(Visual.Property.Type, new PropertyValue((int)Visual.Type.Image))
+               .Add(ImageVisualProperty.URL, new PropertyValue(_imageURL));
+
+    // Setup size of image visual
+    PropertyMap imageVisualTransform = new PropertyMap();
+    imageVisualTransform.Add((int)VisualTransformPropertyType.Offset, new PropertyValue(new Vector2(10, 10)))
+                        .Add((int)VisualTransformPropertyType.OffsetPolicy, new PropertyValue(new Vector2((int)VisualTransformPolicyType.Absolute, (int)VisualTransformPolicyType.Absolute)))
+                        .Add((int)VisualTransformPropertyType.SizePolicy, new PropertyValue(new Vector2((int)VisualTransformPolicyType.Absolute, (int)VisualTransformPolicyType.Absolute)))
+                        .Add((int)VisualTransformPropertyType.Size, new PropertyValue(new Vector2(200, 200)))
+                        .Add((int)VisualTransformPropertyType.Origin, new PropertyValue((int)Visual.AlignType.CenterBegin))
+                        .Add((int)VisualTransformPropertyType.AnchorPoint, new PropertyValue((int)Visual.AlignType.CenterBegin));
+
+    imageVisual.SetTransformAndSize(imageVisualTransform, new Vector2(this.SizeWidth, this.SizeHeight));
+
+    //Create and register image visual.
+    VisualBase imageVisual = VisualFactory.Instance.CreateVisual(map);
+    RegisterVisual(IMAGE_VISUAL_INDEX, imageVisual);
+
+    /// Set the image visual depth index
+    imageVisual.DepthIndex = IMAGE_VISUAL_INDEX;
+    ```
+2. Load image resource in Image `Component`, for more information, see  [ImageView tutorial](imageview.md) page:
```

Comment:
Image `Component`,  

Here I is in capital, please check?

Reply:
In this sentence, does this "Image Component" mean a specific component?
In my understanding, it is not specific and just showing the notion of an image component that loads image resource. - So there is no `Component` in the following code, as the `window` is the image component which is loading the image.

So I'm not sure whether the code tag is required at this `Component`. I think removing the code tag from the `Component` is better.

Reply:
In my understanding, the `components` refers to the namespace: Tizen.NUI.BaseComponents that is used by ImageView class.

@mihashco can you please elaborate on this? This was originally set in a code tag, and I retained it. But can you please see @mijin85cho 's concern?

Thanks.